### PR TITLE
Fix bug in topk_bubble_BLK_on_channel_horiz method

### PR DESCRIPTION
**Pull request by xuchen-intel**
### Details:
 - *Fix bug in topk_bubble_BLK_on_channel_horiz method*

### Tickets:
 - *64906*

### CODE REVIEW

File: src/tests/functional/plugin/cpu/single_layer_tests/topk.cpp [hunk 1]
```diff
@@ -283,6 +283,44 @@ INSTANTIATE_TEST_CASE_P(smoke_TopK_dynamic, TopKLayerCPUTest,
         ::testing::ValuesIn(additionalConfig)),
     TopKLayerCPUTest::getTestCaseName);
 
+std::vector<ov::test::InputShape> inputShapes_bubble_BLK_on_channel_horiz = {
+    {{}, {{2, 2, 2, 2}}},
```
Comment (xuchen-intel) [inline src/tests/functional/plugin/cpu/single_layer_tests/topk.cpp:287]: For topk_bubble_BLK_on_channel_horiz routine in JIT kernel, though we already have test cases with  top_k = 1, axis_dim = 21, which will test this routine. It is still quite necessary to add this corner case with top_k=1, axis_dim=2, where axis_dim < blk_size.

**Comment by xuchen-intel** [comment]
@dmitry-gorokhov Hi Dmitry, could you please take a look?

**Review by dmitry-gorokhov — APPROVED** [review]
